packages/react-ui-ag/src/Banners/__tests__/Banner-test.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+
+import ThemedBanner from '../Banner'
+
+const Banner = ThemedBanner.WrappedComponent
+
+describe('ag/Banners/Banner', () => {
+  it('renders a banner', () => {
```

Comment:
This test and the following should probably be grouped since you are testing `<Text/>` on both of them.